proposals/csharp-12.0/using-alias-types.md

```diff
@@ -22,6 +22,8 @@ using_alias_directive
     ;
 ```
 
+Note: this grammar excludes top-level reference type nullability annotations.
```

Comment:
It's not obvious to me how the grammar can do this, when nullable value types and reference types are syntactically the same.

Reply:
You're right. We need a special rule to exclude nullable reference types. Thanks